design-proposals/monitoring-guidelines.md

```diff
@@ -0,0 +1,39 @@
+## KubeVirt Monitoring
+ 
+### KubeVirt Metrics
+#### Naming a New KubeVirt Metrics:
+
+The KubeVirt metrics should align with the Kubernetes metrics names.
+
+The KubeVirt Users should have the same experience when searching for a node, container, pod and virtual machine metrics.
+
+**Naming requirements:**
+1. Check if a similar Kubernetes metric, for node, container or pod, exists and try to align to it.
+2. KubeVirt metric for a running VM should have a `kubevirt_vmi_` prefix
+
+For Example, see the following Kubernetes network metrics:
+- **node**_network_receive_packets_total
+- **node**_network_transmit_packets_total
+- **container**_network_receive_packets_total
+- **container**_network_transmit_packets_total
+
+The KubeVirt metrics for vmi should be:
+- **kubevirt_vmi**_network_receive_packets_total
+- **Kubevirt_vmi**_network_transmit_packets_total
```

Comment:
It would be good to also describe the list of labels that each metric will have. This helps to better understand the metric coverage.

Reply:
@marceloamaral  There is a separate documentation for the metrics, their description and labels, https://github.com/kubevirt/kubevirt/blob/master/docs/metrics.md.
I created a pr to update it, since currently its corrupted, https://github.com/kubevirt/kubevirt/pull/5404.